raiden/tests/unit/test_transfer.py

```diff
@@ -154,32 +154,37 @@ def test_transfer(raiden_network):
     assert app1.raiden.address in graph0.partneraddress_to_channel
 
     amount = 10
+    identifier = 1337
     target = app1.raiden.address
-    result = app0.raiden.transfer_async(
-        graph0.token_address,
-        amount,
-        target,
-    )
 
-    result.wait(timeout=10)
-    gevent.sleep(5)
+    # assert multiple transfers with same identifier are idempotent
+    for _ in range(2):
```

Comment:
I would prefer a new test instead of repurposing an existing one, with a descriptive name and asserting only on the required results.

Reply:
Ok, will implement it on a new test.